test/e2e/upgrade/upgrade.go

```diff
@@ -365,7 +365,8 @@ func clusterUpgrade(f *framework.Framework, c configv1client.Interface, dc dynam
 			var observedGeneration int64
 
 			// wait until the cluster acknowledges the update
-			if err := wait.PollImmediate(5*time.Second, 2*time.Minute, func() (bool, error) {
+			// Timeout was previously 2min, bumped while work underway on https://bugzilla.redhat.com/show_bug.cgi?id=2071998
+			if err := wait.PollImmediate(5*time.Second, 4*time.Minute, func() (bool, error) {
```

Comment:
If it's just affecting metal, can we make the timeout conditional on platform? We already have the infrastructure CR above

Reply:
Updated, there's also one openstack job hitting so in included that as well, and everything else back to 2min.